src/Turbo/src/Helper/TurboStream.php

```diff
@@ -86,12 +86,28 @@ public static function refresh(?string $requestId = null): string
         return \sprintf('<turbo-stream action="refresh" request-id="%s"></turbo-stream>', htmlspecialchars($requestId));
     }
 
-    private static function wrap(string $action, string $target, string $html, string $attr = ''): string
+    /**
+     * Custom Action.
+     *
+     * @param array<string> $attr
+     */
+    public static function custom(string $action, string $target, string $html, array $attr = []): string
     {
+        return self::wrap($action, $target, $html, $attr);
+    }
+
+    /**
+     * @param array<string> $attr
+     */
+    private static function wrap(string $action, string $target, string $html, array $attr = []): string
+    {
+        // Join array elements with a space and prepend a leading space
+        $atrrString = empty($attr) ? '' : ' '.implode(' ', $attr);
```

Comment:
the API for $attr means people are responsible for encoding their attributes, is that what we want?

Reply:
```suggestion
        $atrrString = $attr ? ' '.implode(' ', $attr) : '';
```

Reply:
> the API for $attr means people are responsible for encoding their attributes, is that what we want?

Not sure, I don't mind either way, but I think it should be documented if users are responsible for encoding.
What do you think ?

Reply:
>  $atrrString = $attr ? ' '.implode(' ', $attr) : '';

I'm not sure I like this. I generally don't like implicit checks, although for empty array it makes sense. I am ok with the implicit check if you insist

Reply:
I think I'd prefer encoding on our side, and $attr be a name=>value list
about the check, empty() is the worst :)